style.css

```diff
@@ -648,7 +601,105 @@ textarea::placeholder {
 
 /* side menu  */
 
+/* popup  */
+.modal-container {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(15px);
+  z-index: 100;
+  display: none;
+  justify-content: center;
+  align-items: center;
+
+  /* display: none; */
+}
+
+.modal {
+  width: auto;
+  max-width: 500px;
+  max-height: 700px;
+  background-color: #fff;
+  padding: 0 20px 20px 20px;
+  margin: 20px;
+  border-radius: 16px;
+  transform: scale(0);
+  transition: all ease-in-out 0.5s;
+  position: relative;
+}
+
+.open-modal {
+  transform: scale(1);
+  transition: all ease-in-out 0.5s;
+}
+
+.open-modal-container {
+  display: flex;
+}
+
+.modal-image img {
+  width: 100%;
+
+  /* max-height: 500px; */
+}
+
```

Comment:
- [ ] Please makes sure that your stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=8QFWKJcYmnQlNLoh-0). As you can see below your modal design on the desktop does not make the Figma design. 

  - Make sure the width of the modal matches the Figma design.
  - Make sure the margins of the margin of modal match the design 
  - The blur color effect does not match the Figma design
  
  ### Your Design 
  ![image](https://user-images.githubusercontent.com/92303891/231779832-3b6963b7-2c94-448e-8f19-be527c111853.png)
  
  ### Figma design
  
  ![Screenshot from 2023-04-13 13-51-58](https://user-images.githubusercontent.com/92303891/231780398-db83a483-30c3-4757-a929-c3aba04ce48d.png)